src/controller_base.cpp

```diff
@@ -17,17 +17,20 @@
 
 #include "display.hpp"
 #include "events.hpp"
+#include "preferences/game.hpp"
 #include "hotkey/command_executor.hpp"
-#include "hotkey/hotkey_command.hpp"
 #include "log.hpp"
 #include "map/map.hpp"
 #include "mouse_handler_base.hpp"
 #include "preferences/game.hpp"
 #include "scripting/plugins/context.hpp"
 #include "show_dialog.hpp" //gui::in_dialog
 #include "soundsource.hpp"
+#include "gui/core/timer.hpp"
+
 static lg::log_domain log_display("display");
 #define ERR_DP LOG_STREAM(err, log_display)
+#define LONG_TOUCH_DURATION_MS 800
```

Comment:
we usually prefer (static) const for such constants instead of macros.

Reply:
Replaced with `static const int`.